apps/passport-client/components/screens/AddScreen/JustAddScreen.tsx

```diff
@@ -105,8 +103,7 @@ export function JustAddScreen({
   } else if (!added) {
     content = (
       <>
-        {isProtocolWorlds && <H2>{"TENSION DISCOVERED".toUpperCase()}</H2>}
-        <Spacer h={16} />
+        {/* {isProtocolWorlds && <H2>{"TENSION DISCOVERED".toUpperCase()}</H2>} */}
```

Comment:
we can just delete everything 'protocol worlds' related in this file and elsewhere